python/jupytercad-app/pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[build-system]
+requires = [
+    "hatchling>=1.5.0",
+    "jupyterlab>=4.0.0,<5",
+    "hatch-nodejs-version>=0.3.2",
+]
+build-backend = "hatchling.build"
+
+[project]
+dependencies = ["jupyter_server>=2.0.1,<3", "jupytercad_core>=0.3.3,<0.4"]
```

Comment:
Are we missing `jupytercad_lab` here? I suppose the `jupytercad_app` package without the `jupytercad_lab` lab plugins would not work?